packages/config/config/devices/0x015f/mh-s510.json

```diff
@@ -0,0 +1,100 @@
+{
+	"manufacturer": "McoHome Technology Co., Ltd.",
+	"manufacturerId": "0x015f",
+	"label": "MH-S510",
+	"description": "Touch Panel Switch",
+	"devices": [
+		{
+			"productType": "0x5131",
+			"productId": "0x5101",
```

Comment:
```suggestion
			"productType": "0xf501",
			"productId": "0x7102",
```

Reply:
Is that from the support? Or did you mean to edit this?
https://github.com/zwave-js/node-zwave-js/pull/3124/files#diff-8917909cccb9b91d0b1261fdfe51094786376bec6cb0b69736176022b50b0d58R8

Reply:
Them are the values reported by the device

Reply:
@blhoward2 Needed them to finish the PR

Reply:
Yeah, but not in the file for the S510?!

Reply:
Ops, sorry that was the wrong file, fixing now